packages/apm/src/integrations/tracing.ts

```diff
@@ -505,6 +572,16 @@ export class Tracing implements Integration {
 
     Tracing._log('[Tracing] Adding & adjusting spans using Performance API');
 
+    // FIXME: depending on the 'op' directly is brittle.
+    if (transactionSpan.op === 'pageload') {
+      // Force any pending records to be dispatched.
+      forceLCP();
+      if (lcp) {
+        // Set the last observed LCP score.
+        transactionSpan.setData('_sentry_extra_metrics', JSON.stringify({ lcp }));
```

Comment:
Sending the LCP as a data entry for now, can be changed.

![image](https://user-images.githubusercontent.com/88819/83314355-4513db00-a21a-11ea-8c92-bf600ccd0e03.png)